pages/dkp/kaptain/2.0.0/install/requirements/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle: Requirements
+title: Requirements
+menuWeight: 3
+excerpt: Requirement reference values
+beta: false
+enterprise: false
+---
+
+The amount of resources required for your project **depends on the workload** you want to run, you need to consider: 
+
+-   The number, complexity, and size of your training jobs, notebook servers, deployed models, etc.
+-   The amount of metadata and log data stored with each run.
+
+To estimate the amount of resources required for your project, it is recommended that you test your specific use case.
+
+To help you in this task, the following table provides an account of the resources required for two common operations: [installing Kaptain][install], and running a complete machine learning workflow using Tensorflow. This workflow corresponds to the [Kaptain SDK with Tensorflow tutorial][tensorflow], and includes the training, tuning, and deploying stages of a machine learning model used to recognize handwritten digits.
+
+The resources in the table are required exclusively for each specified task, and assume you have previously allocated other resources to install and run DKP. Running any real world machine learning workload on Kaptain bumps these requirements. For this reason, the number of CPUs, GPUs, RAM, and persistent disk space **must be increased considerably**. 
+
+Consider the following use cases as reference values:
+
+| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
+|------------------------|----------------------|--------------------------------------|
+| Number of worker nodes | at least 3           | at least 3                           |
+| CPU per node           | 8 cores              | 24 cores                             |
+| RAM per node           | 16 GiB               | 86 GiB                               |
+| Storage per node       | 60 GiB               | 61 GiB                               |
+
```

Comment:
based on @andresmascl's [measurements](https://docs.google.com/spreadsheets/d/1VoF4MCsiUD2NnogcPZ-2wSsco1auZY-j91o_nHZPxMY/edit#gid=1219806816), I can see CPU requests are doubled and memory requests are x4.  Storage increase by 1 GB is rather confusing and can be ignored. Adjusting the numbers to the closest "power of two" values:
```suggestion

| Cluster resource       | Kaptain installation | Kaptain SDK with Tensorflow tutorial |
|------------------------|----------------------|--------------------------------------|
| Number of worker nodes | at least 3           | at least 3                           |
| CPU per node           | 8 cores              | 16 cores                             |
| RAM per node           | 16 GiB               | 64 GiB                               |
| Storage per node       | 60 GiB               | 60 GiB                               |

```